exercises/simple-linked-list/src/example/java/SimpleLinkedList.java

```diff
@@ -0,0 +1,79 @@
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
+/**
+ *
+ * @author matt
+ */
+public class SimpleLinkedList {
+
+    class Element {
+
+        int value;
+        Element next;
+
+        Element(int value) {
+            this.value = value;
+        }
+    }
+
+    private Element head;
+    private int size = 0;
```

Comment:
I went back and checked this out, and turns out the issue with an uninitialized int was later, in the `isArray` method.  I saw it there and changed it everywhere.  Still don't know why it would be a problem when the int is used in an index.